src/app/application.cpp

```diff
@@ -614,8 +614,8 @@ int Application::exec(const QStringList &params)
 #endif // DISABLE_WEBUI
 #else
     UIThemeManager::initInstance();
-    m_window = new MainWindow;
     UIThemeManager::instance()->applyStyleSheet();
+    m_window = new MainWindow;
```

Comment:
This was originally not working because of the setting of the overriding of the global stylesheet in StatusBar.cpp (fixed in the previous commit)

Reply:
Now, I guess we don't need to take `applyStyleSheet()` outside of the `UIThemeManager` constructor anymore, so can we make it private?